src/sentry/sentry_metrics/transactions.py

```diff
@@ -39,6 +39,7 @@ class TransactionTagsKey(Enum):
     """Identifier for a transaction-related tag."""
 
     TRANSACTION_STATUS = "transaction.status"
+    TRANSACTION_SATISFACTION = "satisfaction"
```

Comment:
Tag name in ingestion is [here](https://github.com/getsentry/relay/blob/7d1f750583be495b84f77f597a7c37b591556835/relay-server/src/metrics_extraction/transactions.rs#L305).